packages/client/src/reducers/contacts.js

```diff
@@ -0,0 +1,40 @@
+export const INITIAL_STATE = {
+  contacts: {},
+};
+
+const reducer = (state, action) => {
+  const address = Object.keys(action.payload)[0];
+  const body = action.payload[address];
+  const newContacts = (state.contacts?.[address] ?? []).slice(0);

```

Comment:
Is this line to get the contact for a specific treasury? Suggest renaming to treasuryContacts